index.html

```diff
@@ -712,101 +712,79 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
-    <section id="messaging">
-      <h2>Messaging</h2>
+    <section id="notifications">
+      <h2>Notifications</h2>
       <section class="informative">
         <h3>Introduction</h3>
         <p>
-          This section defines when messages are emitted by a Fedora
-          implementation, the minimal set of data contained in those messages and how
-          those data are serialized. These messages may occur synchronously or asynchronously
-          with the API operations that cause them to be emitted. They are typically used to
-          support external integrations. The structure of these messages draws upon the
+          This section defines when notifications are made available by a Fedora
+          implementation, the minimal set of data contained in these notifications and how
+          the data are serialized. Notifications may be emitted synchronously or asynchronously
+          with the API operations that cause them to be emitted. These notifications are typically
+          used to support external integrations. The structure of these notifications draws upon the
           existing [[activitystreams-core]] and [[ldn]] specifications.
-          An implementation is free to choose from any messaging technology so long as
-          the messages conform to what is described in the following sections.
-        </p>
-      </section>
-
-      <section id="emitting-messages">
-        <h3>Emitting Messages</h3>
-        <p>
-          Any operation that causes the state of a resource to durably change MUST cause a
-          message to be emitted with a corresponding resource identifier.
-        </p>
-        <p>
-          A failed operation (HTTP 4xx or 5xx) MUST NOT emit any messages.
-        </p>
-        <p>
-          Messages MUST NOT be emitted until after any changes have been persisted to durable storage.
+          An implementation is free to choose from any transport technology so long as
+          the notifications conform to what is described in the following sections.
         </p>
         <p>
-          Repository start, stop and configuration change operations MAY cause a message to be emitted.
+          Implementers should be aware that some operations cause multiple resources to change.
+          In these cases, there will be a corresponding notification describing each of the changes.
```

Comment:
@barmintor at the level of event-description (e.g. within an AS Collection or as a singleton notification) the same rules of description apply, namely that (1) it `MUST` conform to the AS 2.0 spec, (2) if a resource has been created/modified/deleted that IRI `MUST` be included and (3) the event type(s) `MUST` be included. w/r/t a separate "event identifier", the suggestion here is to remove that requirement. In so doing, we do not preclude implementations from minting an identifier for such notifications, but I have yet to see a single use case for event identifiers that would require them to be server-generated. 

Requiring behaviors in the absence of use cases is exactly the sort of thing I'd like to avoid with this specification. Likewise, prohibiting behaviors without a _really_ solid justification is what I also want to avoid. Those two principles are what underlie these changes.